<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_file_form</class>
 <widget class="QDialog" name="edit_file_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание / редактирование метаданных файла</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lName">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lDesc">
       <property name="text">
        <string>Описание:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDesc"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lShoot">
       <property name="text">
        <string>Дата и время съемки:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dteShoot"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lRegion">
       <property name="text">
        <string>Регион</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leRegion"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lKA">
       <property name="text">
        <string>Тип КА:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leKA"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="lFilePath">
       <property name="text">
        <string>Файл:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFilePath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbAddFile">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../ddocs_ico/ddocs_icon_set.qrc">
         <normaloff>:/ddoc/find_doc.png</normaloff>:/ddoc/find_doc.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ddocs_ico/ddocs_icon_set.qrc"/>
 </resources>
 <connections/>
</ui>
